codecov.yml

```diff
@@ -21,3 +21,5 @@ ignore:
   - "x/**/test_common.go"
   - "scripts/"
   - "contrib"
+  - "intetgration_test/**/*"
```

Comment:
tbh, I'm not sure if codcov is able to detect our customized integration test coverage for the chain

Reply:
I think overall it does seem to affect the coverage: 
![image](https://github.com/sei-protocol/sei-chain/assets/18161326/0c8ad99c-41a3-4573-844b-af80eaf9c3b6)